gke-deploy/cmd/common/common.go

```diff
@@ -106,6 +106,13 @@ func GcloudInPath() bool {
 	return true
 }
 
+// UseGsutil returns true if the config or suggestedOutput or expandedOutput has a prefix of gs://.
+// This indicates whether to pull the config from GCS or push the output to GCS.
+func UseGsutil(config, output string) bool {
```

Comment:
There are references to `gs://` in deployer, e.g. `gke-deploy/deployer/deployer.go#L54` that can be replaced with a call to this function.

Reply:
This function checks if either `filename` or `output` starts with `gs://`. In which case `gsutil` is needed.

The deployer checks each field `config, expectedOutput,  suggestedOutput` and takes some `gsutil` action.  
It is possible that `config` does not start with `gs://` in which case there is no need to fetch from `GCS`. However, `expected/suggested output` could start with `gs://` in which case we want to push to `GCS`.